### DISCUSSION TITLE
Bug report: editing an issue comment removes the "Sponsor" pill

### DISCUSSION THREAD

**Discussion by strugee** [community/community#77253] [Projects and Issues]
### Select Topic Area

Bug report

### Feature Area

Issues

### Body

STR:

1. Be sponsoring a repository's organization and a contributor to (but not collaborator on) that repository - no idea if this is required, but that's what my status is
2. Comment on an issue
3. Notice that the top-right of your comment box have "Sponsor" and "Contributor" pills
4. Edit your comment
5. Wait a second and notice that the "Sponsor" pill disappears

### Guidelines

- [x] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by strugee** [comment]
Accidentally refreshed and submitted with the wrong category ("Question"), sorry! Can someone adjust the label?

**Comment by queenofcorgis** [comment]
Hi @strugee , we're happy you're here! We've adjusted the label, but you are more likely to get a useful response if you are explicit about what your question entails, giving a few more details might help someone give you a nudge in the right direction.

**Reply by strugee** [reply]
@queenofcorgis so sorry! Looks like the rest of my STR got eaten somehow? Or maybe I just got distracted? I could've _sworn_ I wrote the whole thing... who knows. I've edited the description to have the complete STR now.

It also looks like the labels didn't actually get changed. Can you change the "Question" label to the "Bug report" label?

**Reply by queenofcorgis** [reply]
@strugee thanks for updating - and thanks for clarifying on the label I've changed it over to bug.
